### PR TITLE
Updated public entities

### PR DESCRIPTION
**Pull request by mmadariaga**
Persist PublicEntities and AdministratorRelPublicEntities in PublicEntities db table

#### Type Of Change 
- [X] Small bug fix
- [ ] New feature or enhancement
- [ ] Breaking change

#### Checklist:
- [X] Commits are named and have tag following [commit rules](https://github.com/irontec/ivozprovider/blob/bleeding/doc/dev/en/commits.md)
- [X] Commits are split per component (schema, web/admin, kamusers, agis, ..)
- [X] Changes have been tested locally
- [ ] Fixes an existing issue (Fixes #XXXX) 
- [ ] Upport from existing Pull request #XXXX

### CODE REVIEW

File: schema/DoctrineMigrations/Version20230817114147.php [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    }
+
+    public function up(Schema $schema): void
+    {
+        // this up() migration is auto-generated, please modify it to your needs
+        $this->addSql(
+            'INSERT INTO PublicEntities
+                (iden, fqdn, platform, brand, client, name_en, name_es, name_ca, name_it)
+            VALUES
+                ("PublicEntities", "Ivoz\Provider\Domain\Model\PublicEntity\PublicEntity", 1, 1, 0, "Entity", "Entidad", "Entidad", "Entity")'
```
Comment (Kaian) [inline schema/DoctrineMigrations/Version20230817114147.php:27]: This probably requires doble backlash \\  check after applying the delta

File: schema/DoctrineMigrations/Version20230817114147.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+            . 'WHERE A.restricted = 1 AND A.companyId IS NULL AND P.iden = "PublicEntities"'
+        );
+
+        $this->addSql(
+            'INSERT INTO PublicEntities
+                (iden, fqdn, platform, brand, client, name_en, name_es, name_ca, name_it)
+            VALUES
+                (
+                    "AdministratorRelPublicEntities",
+                    "Ivoz\Provider\Domain\Model\AdministratorRelPublicEntity\AdministratorRelPublicEntity",
```
Comment (Kaian) [inline schema/DoctrineMigrations/Version20230817114147.php:42]: Same here

**Comment by Kaian** [comment]
This probably requires changing the base, it has commits of another PR.

File: schema/DoctrineMigrations/Version20230817114147.php (hunk 1 above)
Comment (mmadariaga) [reply schema/DoctrineMigrations/Version20230817114147.php]: fixed :+1:

**Review by Kaian — APPROVED** [review]

**Pull request merged**